bin/pycbc_live

```diff
@@ -192,7 +192,7 @@ class LiveEventManager(object):
 
 
     def check_coincs(self, ifos, coinc_results, psds, f_low,
-                     data_readers, bank):
+                     data_readers, bank, mc_area_args):
```

Comment:
It seems this `mc_area_args` does not change from trigger to trigger, correct? If so, it makes sense to pass it to the constructor of `LiveEventManager` and store it as an attribute of the `LiveEventManager`, instead of passing it as an argument to `check_coincs()`. Presumably we want to use it in `check_singles()` as well but maybe not in this PR?

Reply:
Yes. Currently is an argument to `check_coincs()` so `check_singles()` doesn't do the source classification, but we can change it in the future

Reply:
Any opinion on my proposal of having `mc_area_args` be an attribute of `LiveEventManager`?

Reply:
I didn't get it right at first, but I think I understood now. Please let me know if I changed it properly

Reply:
Yes, thanks!